### PR TITLE
WRQ-25865: Updated package dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by ion-andrusciac-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated dependencies to the latest minor version.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated dependencies to the latest minor version.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-25865

### Comments

Enact-DCO-1.0-Signed-off-by: Ion Andrusciac ion.andrusciac@lgepartner.com

### CODE REVIEW

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: sample-runner/moonstone/package.json [hunk 1]
```diff
@@ -30,15 +30,15 @@
     "dist/*"
   ],
   "dependencies": {
-    "@enact/core": "^4.7.9",
-    "@enact/i18n": "^4.7.9",
-    "@enact/moonstone": "^4.5.5",
-    "@enact/spotlight": "^4.7.9",
-    "@enact/ui": "^4.7.9",
-    "ilib": "^14.19.0",
+    "@enact/core": "^4.8.0",
```
Comment (hong6316) [inline sample-runner/moonstone/package.json:33]:
```suggestion
    "@enact/core": "~4.8.0",
    "@enact/i18n": "~4.8.0",
    "@enact/moonstone": "^4.5.6",
    "@enact/spotlight": "~4.8.0",
    "@enact/ui": "~4.8.0",
    "ilib": "~14.19.0",
    "prop-types": "^15.8.1",
    "react": "~18.2.0",
    "react-dom": "~18.2.0",
    "react-live": "^4.1.7"
```

**Comment by hong6316** [comment]
Please fix conflicts

File: sample-runner/moonstone/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ion-andrusciac-lgp) [reply sample-runner/moonstone/package.json:33]: We decided to have ~4.8.0 for enact dependencies in the moonstone library. I think leaving ^4.7.9 will lead in the future to the installation of enact 4.9.0 and will cause functional issues. Do you agree that we should put ~4.8.0?

[the remark this answers is not in the document]
Comment (hong6316) [reply sample-runner/moonstone/package.json:33]: I see, agree.
Please fix them to `~4.8.0`, and fix ilib `~14.19.0`

**Pull request merged**